ykcs11/ykcs11.c

```diff
@@ -536,6 +536,7 @@ CK_DEFINE_FUNCTION(CK_RV, C_OpenSession)(
     session.info.state = CKS_RO_PUBLIC_SESSION; // Nobody has logged in, default RW session
   }
 
+  session.info.slotID = slotID;
```

Comment:
This is already at line 525 but is commented out. We might as well uncomment that line.